srs_extension/extension.js

```diff
@@ -105,6 +105,80 @@ function activate(context) {
         }
     };
 
+    var insertReqTags = function () {
+        var text = loadText();
+
+        if (text) {
```

Comment:
>if (text) { [](start = 8, length = 11)

Maybe there should be a method that gets as arg the input text, returns the text as it should be pasted in the editor.
Then it would be testable.

Reply:
At least we should add an issue with this so we can track it somehow ...

---
In reply to: [405873254](https://github.com/Azure/azure-c-build-tools/pull/18#discussion_r405873254) [](ancestors = 405873254)

Reply:
Or even better maybe you can refactor the code a bit now to have that method even if you do not add all the tests for it now ... Althoughonce refactored my feeling is that adding some tests might be very easy :-).

---
In reply to: [405873414](https://github.com/Azure/azure-c-build-tools/pull/18#discussion_r405873414) [](ancestors = 405873414,405873254)

Reply:
Int tests added (did not refactor this for now)

---
In reply to: [405873662](https://github.com/Azure/azure-c-build-tools/pull/18#discussion_r405873662) [](ancestors = 405873662,405873414,405873254)